plugin-tester-java/src/main/java/example/myapp/statefulhelloworld/GreeterServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package example.myapp.statefulhelloworld;
+
+import example.myapp.statefulhelloworld.grpc.*;
+
+import akka.actor.ActorSystem;
+import akka.actor.ActorRef;
+import akka.util.Timeout;
+import static akka.pattern.PatternsCS.ask;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+// #stateful-service
+public final class GreeterServiceImpl implements GreeterService {
+
+  private final ActorSystem system;
```

Comment:
I guess `system` doesn't even need to be a field here - though perhaps good to show since real-life implementations might often need it?

Reply:
Yeah, actually I thought I'd need it for ask, but then CS-ask doesn't. Let's keep it anyway